python/tchannel/tornado/tornado_handler.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+from tornado import httputil
+from tornado.httputil import RequestStartLine
+from ..handler import RequestHandler
+import tornado.httputil
+
+from http_request import HttpRequest
+tornado.httputil.HTTPServerRequest = HttpRequest
+
+
+class TornadoRequestHandler(RequestHandler):
+    def __init__(self, app):
+        self.request_callback = app
+
+    def start_serving(self, request_conn):
+        return _ServerRequestAdapter(self, request_conn)
+
+    def handle_request(self, context, conn):
+        """dispatch incoming request to particular endpoint
+
+        :param context: context contains received CallRequestMessage
+        :param conn: An incoming TornadoConnection
+        """
+        request_delegate = self.start_serving(conn)
+        message = context.message
+        # process http message
+        """TODO need a better way to figure out
+        message's method type
+        """
+        if message.headers["as"] == "http":
+            method = "GET"
+            if (message.arg_3 is not None and
+                    message.arg_3 != ""):
+                method = "POST"
+
+            start_line = RequestStartLine(method, message.arg_1, 'HTTP/1.1')
+            try:
+                headers = json.loads(message.arg_2)
+            except:
+                headers = {}
+            body = message.arg_3 if hasattr(message, "arg_3") else ""
+            request_delegate.headers_received(start_line, headers)
+            request_delegate.data_received(body)
+            request_delegate.finish()
+        # TODO process message in json/thrift format
```

Comment:
Generally when leaving TODOs like this, add a `raise NotImplementedError()` so that this isn't missed.